docs/deployment/on-chain_deployment_checklist.md

```diff
@@ -203,6 +203,18 @@ Ensure that your components, agent and service packages are published to the IPF
 Ensure that your components, agent and service packages are registered in the on-chain protocol. You can [use the protocol front-end](../guides/register_packages_on_chain.md) or use the `autonomy mint` command.
 
 
+## Check the deployment readiness of the service using
+
+```
+$ autonomy analyse service PUBLIC_ID
+```
+
+or if you want to check deployment readiness of an on-chain service
+
+```
+$ autonomy analyse service PUBLIC_ID --token-id TOKEN_ID
```

Comment:
Why public id if token id is provided? Makes no sense

Reply:
Also once a service is on-chain it is presummably already verified?

Reply:
On chain check is to make sure that every package required is published on the IPFS registry

Reply:
To address your first comment https://github.com/valory-xyz/open-autonomy/pull/1829